src/Explorer/Notebook/useNotebook.ts

```diff
@@ -309,7 +310,11 @@ export const useNotebook: UseStore<NotebookState> = create((set, get) => ({
 
       const isPublicInternetAllowed = isPublicInternetAccessAllowed();
       const phoenixClient = new PhoenixClient(userContext?.databaseAccount?.id);
-      const dbAccountAllowedInfo = await phoenixClient.getDbAccountAllowedStatus();
+      const dbAccountAllowedInfo = await promiseRetry(() => phoenixClient.getDbAccountAllowedStatus(), {
+        retries: 10,
```

Comment:
Any reason for having 10 retries?

Reply:
Also how do we differentiate between 403 because of ARM Call or 403 because of not having ListKey Permissions/RegionNotServicable which are valid?

Reply:
May be you could exclude the known cases and abort the retry in those cases? Like if 403 is because of 
1)RegionNotServicable
2)SubscriptionNotAllowed
 You can do something like [throw new AbortError(error)], other cases you may retry?

Reply:
Discussed offline with Karthik